client/build.gradle

```diff
@@ -1,16 +1,14 @@
 plugins {
     id 'com.gorylenko.gradle-git-properties' version '1.4.16'
-    id 'application'
-}
-application {
-    mainClassName = 'com.github.jsappz.lamegame.game.Main'
 }
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
+    // https://mvnrepository.com/artifact/commons-io/commons-io
+    compile group: 'commons-io', name: 'commons-io', version: '2.6'
```

Comment:
Probably  a follow up to add this to dependencies.gradle instead @JReistad

Reply:
will write up a ticket for it